source/guide/3.2/reference-cli-tasks.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: bt_wiki
+title: CLI Tasks Reference
+category: Reference
+publish: true
+abstract: "high level information on the Cloudify CLI tasks"
+pageord: 900
+---
+
+{%summary%}
+These tasks are used in [Manager blueprints](reference-terminology.html#manager-blueprints) to bootstrap a Cloudify Manager.
+This page will give a high level description of them.
+{%endsummary%}
+
+
+# bootstrap
+
+## Overview
+
+This bootstrap task uses the Fabric plugin to bootstrap a Cloudify Manager.
+
+It's currently a part of the CLI code, and can be found at / mapped to: `cloudify_cli.bootstrap.tasks.bootstrap`.
+
+The task does more or less the following:
```

Comment:
I'm not sure it's a good idea to phrase it like this in docs (sounds very shaky)